packages/cli/src/lib/cli.spec.ts

```diff
@@ -1,10 +1,12 @@
-import { describe, expect, it } from 'vitest';
-import { yargsCli } from './cli';
-import { join } from 'path';
-import { yargsGlobalOptionsDefinition } from './options';
-import { middlewares } from './middlewares';
-import { CommandBase } from './implementation/base-command-config';
+import {describe, expect, it} from 'vitest';
```

Comment:
not sure if the prettier is failing, or you don't run it in webstorm on save action but the import {} format is wrong 😂

Reply:
Fixed.